### PR TITLE
i18n(zh-CN): Update getting-started.mdx

### PR DESCRIPTION
**Pull request by leovoon**
#### What kind of changes does this PR include?
-  Changes or translations of Starlight docs site content

#### Description

-  Update for https://github.com/withastro/starlight/pull/1026

### CODE REVIEW

File: docs/src/content/docs/zh-cn/getting-started.mdx [hunk 1]
```diff
@@ -45,24 +46,69 @@ yarn create astro --template starlight
...
+### 添加内容
+
+Starlight已经准备好让你添加新内容或导入你现有的文件！
+
+### 文件格式
+
+Starlight支持在Markdown和MDX中编写内容，无需进行任何配置。
+你可以通过安装实验性的[Astro Markdoc](https://docs.astro.build/zh-cn/guides/integrations-guide/markdoc/)集成来添加对Markdoc的支持。
+
+### 添加页面
```
Comment (delucis) [inline docs/src/content/docs/zh-cn/getting-started.mdx:91]: I think these two headings should be level 4 like in the English version:

```suggestion
#### 文件格式

Starlight支持在Markdown和MDX中编写内容，无需进行任何配置。
你可以通过安装实验性的[Astro Markdoc](https://docs.astro.build/zh-cn/guides/integrations-guide/markdoc/)集成来添加对Markdoc的支持。

#### 添加页面
```

File: docs/src/content/docs/zh-cn/getting-started.mdx [hunk 2]
```diff
@@ -93,37 +142,35 @@ Starlight 是一个 Astro 集成，可以像任何 `@astrojs/*` 集成一样更
...
 
 ## Starlight 故障排除
 
 本站点的参考部分提供了 Starlight [项目配置](/zh-cn/reference/configuration/)和[单个页面 frontmatter 配置](/zh-cn/reference/frontmatter/)信息。使用这些页面来确保你的 Starlight 网站已正确配置和运行。
 
-请参阅侧边栏中不断增长的指南列表，以获取有关添加内容和自定义 Starlight 网站的帮助。
+请参阅侧边栏中的指南列表，以获取有关添加内容和自定义 Starlight 网站的帮助。
 
-如果你在这些文档中找不到答案，请访问[完整的 Astro 文档](https://docs.astro.build) 以获取完整的 Astro 文档。你的问题可能是通过了解 Starlight 主题下 Astro 的工作原理来解决的。
+如果你在这些文档中找不到答案，请访问[完整的 Astro 文档](https://docs.astro.build) 以获取完整的 Astro 文档。
```
Comment (delucis) [inline docs/src/content/docs/zh-cn/getting-started.mdx:173]: This link was already like this, but I think we can fix it to use `/zh-cn/`:

```suggestion
如果你在这些文档中找不到答案，请访问[完整的 Astro 文档](https://docs.astro.build/zh-cn/) 以获取完整的 Astro 文档。
```

File: docs/src/content/docs/zh-cn/getting-started.mdx [hunk 3]
```diff
@@ -45,24 +46,69 @@ yarn create astro --template starlight
 [在 StackBlitz 上打开模板](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics)。
 :::
 
-## 使用 Starlight 创建内容
+### 启动开发服务器
 
-Starlight 已经准备好让你添加新的内容，或者将你现有的文件带过来！
+在本地工作时，Astro的开发服务器允许你预览你的工作，并在你进行更改时自动刷新你的浏览器。
```
Comment (delucis) [inline docs/src/content/docs/zh-cn/getting-started.mdx:51]: In the English version of this paragraph, the text “Astro’s development server” links to the Astro docs: https://docs.astro.build/zh-cn/reference/cli-reference/#astro-dev

Can we add the link here too?

**Review by delucis — COMMENTED** [review]
Thank you for the translation @leovoon! Left a couple of comments for things we could improve.

File: docs/src/content/docs/zh-cn/getting-started.mdx (hunk 3 above)
Comment (leovoon) [reply docs/src/content/docs/zh-cn/getting-started.mdx]: Ouch, careless. I have updated it.

**Review by delucis — APPROVED** [review]
Thanks for making those fixes! 🚀

**Pull request merged**